modules/social_features/social_group/social_group.api.php

```diff
@@ -181,6 +181,18 @@ function hook_social_group_allowed_join_method_description_alter($key, &$descrip
   }
 }
 
+/**
+ * Adds extra group types with visibility fields that work as flexible groups.
```

Comment:
@rollki Please add a better description, so it will be clear for developers how they can use this hook. Something like:

> Provide a method to extend group and content visibilities to other group types.